resources/lang/en/patrols/new_cat.json

```
@@ -4327,5 +4327,393 @@
                 "outsider_rep": -2
             }
         ]
+    },
+    {
+        "patrol_id": "gen_train_app_newfriend1",
+        "biome": ["any"],
+        "season": ["any"],
+        "types": ["training"],
+        "tags": ["new_cat"],
+        "patrol_art": "train_general_intro",
+        "min_cats": 1,
+        "max_cats": 1,
+        "min_max_status": {
+            "apprentice": [1, 1]
+        },
+        "weight": 20,
+        "intro_text": "p_l is out doing some solo training when, suddenly, {PRONOUN/p_l/subject} {VERB/p_l/spot/spots} another cat around {PRONOUN/p_l/poss} age.",
```

This doesn't quite read comfortably to me - moving the clause to get rid of the extra commas feels more natural?
```suggestion
        "intro_text": "p_l is out doing some solo training when {PRONOUN/p_l/subject} suddenly {VERB/p_l/spot/spots} another cat around {PRONOUN/p_l/poss} age.",
```